setup.py

```diff
@@ -23,8 +23,10 @@
         'acl_loader',
         'clear',
         'clear.plugins',
```

Comment:
Looks like the below PR also has the same changes for some files, are we planning to consolidate the changes in one PR?

https://github.com/Azure/sonic-utilities/pull/1650/files

Reply:
Changes from this PR was added to [https://github.com/Azure/sonic-utilities/pull/1650/files](https://github.com/Azure/sonic-utilities/pull/1650/files) in order to pass CI, once this PR will be merged, the changes will be removed from [https://github.com/Azure/sonic-utilities/pull/1650/files](https://github.com/Azure/sonic-utilities/pull/1650/files)